llvm/lib/Support/APFloat.cpp

```diff
@@ -130,17 +132,19 @@ static constexpr fltSemantics semIEEEsingle = {127, -126, 24, 32};
 static constexpr fltSemantics semIEEEdouble = {1023, -1022, 53, 64};
 static constexpr fltSemantics semIEEEquad = {16383, -16382, 113, 128};
 static constexpr fltSemantics semFloat8E5M2 = {15, -14, 3, 8};
-static constexpr fltSemantics semFloat8E5M2FNUZ = {
-    15, -15, 3, 8, fltNonfiniteBehavior::NanOnly, fltNanEncoding::NegativeZero};
-static constexpr fltSemantics semFloat8E4M3FN = {
-    8, -6, 4, 8, fltNonfiniteBehavior::NanOnly, fltNanEncoding::AllOnes};
-static constexpr fltSemantics semFloat8E4M3FNUZ = {
-    7, -7, 4, 8, fltNonfiniteBehavior::NanOnly, fltNanEncoding::NegativeZero};
-static constexpr fltSemantics semFloat8E4M3B11FNUZ = {
-    4, -10, 4, 8, fltNonfiniteBehavior::NanOnly, fltNanEncoding::NegativeZero};
+// clang-format off
+static constexpr fltSemantics semFloat8E5M2FNUZ = {15,-15,3,8,APFloatBase::BaseTwo,fltNonfiniteBehavior::NanOnly,fltNanEncoding::NegativeZero};
+static constexpr fltSemantics semFloat8E4M3FN = {8,-6,4,8,APFloatBase::BaseTwo,fltNonfiniteBehavior::NanOnly,fltNanEncoding::AllOnes};
+static constexpr fltSemantics semFloat8E4M3FNUZ = {7,-7,4,8,APFloatBase::BaseTwo,fltNonfiniteBehavior::NanOnly,fltNanEncoding::NegativeZero};
+static constexpr fltSemantics semFloat8E4M3B11FNUZ = {4,-10,4,8,APFloatBase::BaseTwo,fltNonfiniteBehavior::NanOnly,fltNanEncoding::NegativeZero};
```

Comment:
I'm happy with disabling clang-format here, but I think we should still try to avoid excessively long lines. Also, spaces after commas, please.
```suggestion
static constexpr fltSemantics semFloat8E5M2FNUZ =
    {15, -15, 3, 8, APFloatBase::BaseTwo, fltNonfiniteBehavior::NanOnly, fltNanEncoding::NegativeZero};
static constexpr fltSemantics semFloat8E4M3FN =
    {8, -6, 4, 8, APFloatBase::BaseTwo, fltNonfiniteBehavior::NanOnly, fltNanEncoding::AllOnes};
static constexpr fltSemantics semFloat8E4M3FNUZ =
    {7, -7, 4, 8, APFloatBase::BaseTwo, fltNonfiniteBehavior::NanOnly, fltNanEncoding::NegativeZero};
static constexpr fltSemantics semFloat8E4M3B11FNUZ =
    {4, -10, 4, 8, APFloatBase::BaseTwo, fltNonfiniteBehavior::NanOnly, fltNanEncoding::NegativeZero};
```